certification/internal/shell/imagesource_registry.go

```diff
@@ -0,0 +1,72 @@
+package shell
+
+import (
+	"bytes"
+	"strings"
+
+	cmdchain "github.com/rainu/go-command-chain"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+	log "github.com/sirupsen/logrus"
+)
+
+type ImageSourceRegistryCheck struct {
+}
+
+func (p *ImageSourceRegistryCheck) Validate(bundleImage string) (bool, error) {
```

Comment:
Please add unit tests :)

Reply:
I just pushed my changes and includes a test.